copter/source/docs/traditional-helicopter-swashplate-setup.rst

```diff
@@ -155,9 +155,45 @@ that produces zero thrust. In Copter 3.6 and ealier, the minimum collective pitc
 could command was done with the parameter ``H_COL_LAND_MIN``. 
 
 Set :ref:`H_SV_MAN <H_SV_MAN>` to 3 to have the swashplate move to the mid position.  Adjust :ref:`H_COL_MID <H_COL_MID>` until the blade
-pitch angle measures the desired collective blade pitch that corresponds to zero thrust.  
+pitch angle measures the desired collective blade pitch that corresponds to zero thrust.
 
+.. note:: If the minimum or maximum collective is measured in ACRO (see below for impact of ACRO on collective pitch curve vs stabilized modes), the value will be less than those since some servo movement is reserved for cyclic. Of course, any swashplate tilt due to pilot or autopilot input will interefere with the measurements also.
 
+ACRO versus STABILIZED Collective Curves
+========================================
+
+ArduPilot provides a programmable curve for stabilized modes (all except ACRO). This curve is set by the ``IM_STB_COL_x`` parameters as shown in the graph below. Only ACRO mode uses the collective extremes that were setup above.
+
+.. image:: ../images/collective_curves.png
+   :target: ../_images/collective_curves.png
+
+By default, these settings mirror the ACRO mode curve, and are satisfactory if the -2° to +10° range is setup. This will give the following values:
+
+==============      =====
+Throttle Stick      Pitch
+==============      =====
+Low                 -2
+Mid                 +4 (close to hover)
+High                +10
+==============      =====
+
+
+However, if the extremes were setup for full aerobatic, the defaults for ``IM_STB_COL_x`` would yield:
+
+==============      =====
+Throttle Stick      Pitch
+==============      =====
+Low                 -10
+Mid                 +0
+High                +10
+==============      =====
+
+which is fine for ACRO, but the auto-collective controlled modes, like AUTO and HOVER, would be overly sensitive to collective. Therefore it is recommended to change the values from default to something like:
```

Comment:
Hover?  This is not a mode. Also the stabilize collective curve is not used for auto-collective modes. The use the full collective range. They are set to the reduced range for an aerobatic setup because stabilize is a 2D mode and doesn’t need that much negative collective.  It is really that the full range is too sensitive for the pilot

Reply:
sorry ALTHOLD (plane guy)....and I am confused by the contradiction in "They are set to the reduced range for an aerobatic setup because stabilize is a 2D mode and doesn’t need that much negative collective"...

Reply:
The stabilize collective curve is provided so the pilot can set a non aerobatic collective range in stabilize mode. That all it is there for.

Reply:
okay so no problem using the full range for aerobatic setups for all the non pilot controlled collective modes....assumed that the softer (in aero setups) stabilize range would be used....will redo the explanations...but does DRIFT (which is like STABILIZE with extras) work correctly ? ie use STABILIZE collective curve?

Reply:
fixed